Realm.xcodeproj/project.pbxproj

```diff
@@ -460,6 +460,21 @@
+		CF56B09C24894E7F0001251D /* remote_mongo_collection.cpp in Sources */ = {isa = PBXBuildFile; fileRef = 498C7F3B24479905009DE80A /* remote_mongo_collection.cpp */; };
+		CF56B09D24894E7F0001251D /* remote_mongo_database.cpp in Sources */ = {isa = PBXBuildFile; fileRef = 498C7F3A24479905009DE80A /* remote_mongo_database.cpp */; };
+		CF56B0A2248950890001251D /* RLMUpdateResult_Private.hpp in Headers */ = {isa = PBXBuildFile; fileRef = CF76F7CE24816AA900890DD2 /* RLMUpdateResult_Private.hpp */; settings = {ATTRIBUTES = (Private, ); }; };
+		CF56B0A3248A8AB70001251D /* RLMUpdateResult_Private.hpp in Headers */ = {isa = PBXBuildFile; fileRef = CF76F7CE24816AA900890DD2 /* RLMUpdateResult_Private.hpp */; settings = {ATTRIBUTES = (Private, ); }; };
+		CF56B0A4248A8ACE0001251D /* RLMMongoDatabase_Private.hpp in Headers */ = {isa = PBXBuildFile; fileRef = CF76F7DC24816AAB00890DD2 /* RLMMongoDatabase_Private.hpp */; settings = {ATTRIBUTES = (Private, ); }; };
+		CF56B0A5248A8ACE0001251D /* RLMMongoDatabase_Private.hpp in Headers */ = {isa = PBXBuildFile; fileRef = CF76F7DC24816AAB00890DD2 /* RLMMongoDatabase_Private.hpp */; settings = {ATTRIBUTES = (Private, ); }; };
+		CF56B0A6248A8AD40001251D /* RLMMongoCollection_Private.hpp in Headers */ = {isa = PBXBuildFile; fileRef = CF76F7D224816AAA00890DD2 /* RLMMongoCollection_Private.hpp */; settings = {ATTRIBUTES = (Private, ); }; };
+		CF56B0A7248A8AD40001251D /* RLMMongoCollection_Private.hpp in Headers */ = {isa = PBXBuildFile; fileRef = CF76F7D224816AAA00890DD2 /* RLMMongoCollection_Private.hpp */; settings = {ATTRIBUTES = (Private, ); }; };
+		CF56B0A8248A8ADA0001251D /* RLMMongoClient_Private.hpp in Headers */ = {isa = PBXBuildFile; fileRef = CF76F7D324816AAA00890DD2 /* RLMMongoClient_Private.hpp */; settings = {ATTRIBUTES = (Private, ); }; };
+		CF56B0A9248A8ADB0001251D /* RLMMongoClient_Private.hpp in Headers */ = {isa = PBXBuildFile; fileRef = CF76F7D324816AAA00890DD2 /* RLMMongoClient_Private.hpp */; settings = {ATTRIBUTES = (Private, ); }; };
+		CF56B0AA248A8AE00001251D /* RLMFindOptions_Private.hpp in Headers */ = {isa = PBXBuildFile; fileRef = CF76F7D424816AAA00890DD2 /* RLMFindOptions_Private.hpp */; settings = {ATTRIBUTES = (Private, ); }; };
+		CF56B0AB248A8AE00001251D /* RLMFindOptions_Private.hpp in Headers */ = {isa = PBXBuildFile; fileRef = CF76F7D424816AAA00890DD2 /* RLMFindOptions_Private.hpp */; settings = {ATTRIBUTES = (Private, ); }; };
+		CF56B0AC248A8AE40001251D /* RLMFindOneAndModifyOptions_Private.hpp in Headers */ = {isa = PBXBuildFile; fileRef = CF76F7D824816AAB00890DD2 /* RLMFindOneAndModifyOptions_Private.hpp */; settings = {ATTRIBUTES = (Private, ); }; };
+		CF56B0AD248A8AE40001251D /* RLMFindOneAndModifyOptions_Private.hpp in Headers */ = {isa = PBXBuildFile; fileRef = CF76F7D824816AAB00890DD2 /* RLMFindOneAndModifyOptions_Private.hpp */; settings = {ATTRIBUTES = (Private, ); }; };
```

Comment:
Private headers should not be part of the target. Membership in the target determines whether the header is included in the built framework and that should only have public headers.